src/javascript/app/pages/user/view_popup/view_popup.js

```diff
@@ -554,26 +557,45 @@ const ViewPopup = (() => {
     };
 
     const populateAuditTable = (show_audit_table) => {
-        const contract_starts = createAuditTable(localize('Contract Starts'));
-        parseAuditResponse(contract_starts.table, contract.audit_details.contract_start).then(() => {
-            if (contract.audit_details.contract_start) {
-                createAuditHeader(contract_starts.table);
-                appendAuditLink('trade_details_entry_spot');
-            } else {
-                contract_starts.div.remove();
-            }
-            // don't show exit tick information if missing or manual sold
-            if (contract.audit_details.contract_end && contract.status !== 'sold') {
-                parseAuditResponse(contract_starts.table, contract.audit_details.contract_end).then(() => {
-                    if (contract.audit_details.contract_end) {
-                        appendAuditLink('trade_details_current_spot');
-                    }
+        if (!is_tick_contract) {
```

Comment:
can we simply check if `contract.tick_count` instead of making a new global variable?